edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/CbsNode.cs

```diff
@@ -289,23 +306,22 @@ Task SendResponseAsync(AmqpMessage requestMessage, AmqpResponseStatusCode status
             }
         }
 
-        /// <summary>
-        /// This type is deliberately mutable because of the use case.
-        /// </summary>
-        class CredentialsInfo
+        internal class ClientToken
```

Comment:
Internal for tests?

Reply:
yea used by one of the unit tests

---
In reply to: [489204579](https://github.com/Azure/iotedge/pull/3552#discussion_r489204579) [](ancestors = 489204579)